### DISCUSSION TITLE
Get Document Text for AC1014 documents

### DISCUSSION THREAD

**Discussion by jefrank0701** [DomCR/ACadSharp#279] [Q&A]
How do I get text from AC1014 documents? When I look at the entities for these documents, the count is zero.

**Comment by DomCR** [comment]
Hi @jefrank0701 

Could you provide a file as an example where you are having this issue?

Thanks

**Comment by jefrank0701** [comment]
Thanks for the reply. I'm using the example program in Visual Studio 2022.  

![Screenshot 2024-02-17 101357](https://github.com/DomCR/ACadSharp/assets/154451798/3033926a-7529-49ea-b144-8d82e538cfbc)

The screenshot is from the locals window from the document object that I opened. It shows that there are zero entities. It also has a collection in _cadObjects that seems to contain entities. 

Please let me know if you need any more information.
Thanks!

**Reply by DomCR** [reply]
I've just tested my own dwg sample and seems that for the version AC1014 is not loading the entities in the right block, I'll open a branch to take a look.
